aztec/src/main/kotlin/org/wordpress/aztec/EnhancedMovementMethod.kt

```diff
@@ -89,4 +94,15 @@ object EnhancedMovementMethod : ArrowKeyMovementMethod() {
 
         return super.onTouchEvent(widget, text, event)
     }
+
+    private fun handleTaskListClick(text: Spannable, off: Int): Boolean {
+        val clickedList = text.getSpans(off, off, AztecTaskListSpan::class.java).firstOrNull()
+        val clickedLine = text.getSpans(off, off, AztecListItemSpan::class.java).lastOrNull()
+        val spanStart = text.getSpanStart(clickedLine)
+        if (spanStart == off && clickedList != null && clickedLine != null && clickedList.canToggle()) {
```

Comment:
the `spanStart == off` here makes sure that we only handle clicks that are before the line start. This means we'll only handle clicks on the checkboxes and around them.